Documentation/EtcdLease.md

```diff
@@ -0,0 +1,69 @@
+# Overview
+
+The EtcdLease implement interface `grant` and `revoke` to new a lease and revoke a lease to etcd.
```

Comment:
The EtcdLease interface provides methods to grant, revoke, and keepalive leases.